enterprise/server/test/integration/podman/podman_test.go

```diff
@@ -380,9 +385,13 @@ func TestForceRoot(t *testing.T) {
 }
 
 func TestUser(t *testing.T) {
+	if runtime.GOARCH == "arm64" {
+		// TODO: build podman ourselves, and remove this
+		t.Skipf("--passwd arg is not yet supported by podman version available on GitHub actions runner")
```

Comment:
Nit: specify the expected vs actual version number so we can flip this in the future when newer versions become available.

Reply:
Done